src/OrchardCore.Modules/OrchardCore.Security/Settings/SecuritySettings.cs

```diff
@@ -0,0 +1,15 @@
+namespace OrchardCore.Security
+{
+    public class SecuritySettings
```

Comment:
Could be replaced by `SecurityHeadersOptions`?

Reply:
Yep!